ci/verify.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+./vendor/bin/php-cs-fixer fix ./src
+./vendor/bin/psalm
+./vendor/bin/phpcs --config-set colors 1
```

Comment:
phpcs doesn't work this way - when you use `--config-set` param, executable changes settings and thats all (returns success code and doesnt perform any checks as you'd expect)